presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -517,6 +517,13 @@ public PlanOptimizers(
                 costCalculator,
                 new TranslateExpressions(metadata, sqlParser).rules()));
         // After this point, all planNodes should not contain OriginalExpression
+        builder.add(new IterativeOptimizer(
```

Comment:
I would expect the rule to be moved rather than added?

Reply:
It is moved up. But the git diff will show as delete + add.

Reply:
I see this commit shows up differently, let me fix that.